IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_ImpactProtectionDeviceOccurrenceBumper/DocPropertySet.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_ImpactProtectionDeviceOccurrenceBumper" UniqueId="cf9e5196-0da4-4b1f-a866-8b95fd297478" PropertySetType="PSET_OCCURRENCEDRIVEN">
```

Comment:
Missing documentation and Applicability?

Reply:
Updated: https://github.com/bSI-InfraRoom/IFC-Specification/pull/168/commits/791b52ae8b05f2e9592ef4fdee6e010917dd44e1